src/components/member-page/donation.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import EmptyState from '@twreporter/react-components/lib/empty-state'
+import { H3 } from '@twreporter/react-components/lib/text/headline'
+import { P2 } from '@twreporter/react-components/lib/text/paragraph'
+import {
+  BRANCH,
+  BRANCH_PROP_TYPES,
+} from '@twreporter/core/lib/constants/release-branch'
+
+const Block = styled.div`
+  height: 72px;
+`
+
+const MemberDonationPage = ({ releaseBranch = BRANCH.master }) => {
+  return (
+    <div>
+      <H3 text={'贊助紀錄'} />
+      <Block />
+      <EmptyState
+        style={EmptyState.Style.UNDER_CONSTRUCTION}
+        title={'功能即將推出！'}
```

Comment:
Question: how do we decide to pass string with {} or ""
(may we use the same method in this file)